includes/abbreviations.md

```diff
@@ -28,3 +30,36 @@
 *[URL]: Uniform Resource Locator
 *[UTC]: Coordinated Universal Time
 *[VM]: Virtual Machine
+
+<!-- Grab description string from config options typescript file, and copy/paste it into the right format here -->
+
+*[detectGlobalManagerConfig]: If `true`, Renovate tries to detect global manager configuration from the file system.
+*[detectHostRulesFromEnv]: If `true`, Renovate tries to detect host rules from environment variables.
+*[allowPostUpgradeCommandTemplating]: Set this to `true` to allow templating for post-upgrade commands.
+*[allowedPostUpgradeCommands]: A list of regular expressions that decide which post-upgrade tasks are allowed.
+*[postUpgradeTasks]: Post-upgrade tasks that are executed before a commit is made by Renovate.
+*[commands]: A list of post-upgrade commands that are executed before a commit is made by Renovate.
```

Comment:
This is possibly a bit too generic to make into a hover any time it's mentioned. We may need to exclude some more common words, e.g. timeout too

Reply:
Yeah we should filter out common words.

Also the hover test does not appear on `monospaced` terms. Basically all mentions of config options are `monospaced`. 🙂 Maybe we can override the plug-in, but that may cause problems with copy/pasting config examples from the docs manually.